README.md

```diff
@@ -58,6 +58,33 @@ the following environment variables to your container:
   - `DB_USER` (default: `root`)
   - `DB_PASS`
 
+### Postgres
+
+In addition to the linked MySQL backend, you can alternatively use a Postgres
```

Comment:
'addition' and 'alternatively' are somehow redundant and paradox at the same time. i propose:
'You can alternatively use a …'